backend/data/extract_covid19_india_data.py

```diff
@@ -80,6 +80,8 @@ def process(self, json_dict: dict) -> (pd.DataFrame, pd.DataFrame):
                         district_dict['state'] = state
                         if district.strip().lower()=='unknown':
                             district_dict['district'] = unknown_dict[state]
+                        elif district.strip().lower() in ["other state", "pratapgarh", "hamirpur", "balrampur", "aurangabad", "bilaspur"]:
+                            district_dict['district'] = district + " - " + state
```

Comment:
Just needs a comment perhaps the title of this pull request would be the perfect comment here. :-)